src/uniprotkb/components/entry/EntryPublications.tsx

```diff
@@ -137,7 +137,7 @@ const EntryPublications: FC<{ accession: string }> = ({ accession }) => {
   return (
     <section>
       <h2>Publications for {accession}</h2>
-      <DataListWithLoader<LiteratureResultsAPI>
+      <DataListWithLoader<CitationsAPIModel>
```

Comment:
```suggestion
      <DataListWithLoader
```
I know it was there before, but I think in this case this might not be needed as `allResults` used in the `data` prop is already strongly defined.